releases.json

```diff
@@ -1,90 +1,90 @@
+            {
+                "device": "gpu",
+                "min_version": "1.4.2",
+                "max_version": "2.0.1",
+                "os_version": "ubuntu22.04",
+                "cuda_version": "cu121",
+                "python_version": "py310",
+                "pytorch_version": "2.1.1"
+            },
+            {
+                "device": "gpu",
+                "min_version": "2.0.2",
+                "max_version": "2.0.3",
+                "os_version": "ubuntu22.04",
+                "cuda_version": "cu121",
+                "python_version": "py310",
+                "pytorch_version": "2.3.0"
+            },
+            {
+                "device": "inf2",
+                "min_version": "0.0.16",
+                "max_version": "0.0.21",
+                "os_version": "ubuntu22.04",
+                "python_version": "py310",
+                "pytorch_version": "1.13.1"
+            },
+            {
+                "device": "inf2",
+                "min_version": "0.0.22",
+                "max_version": "0.0.23",
+                "os_version": "ubuntu22.04",
+                "python_version": "py310",
+                "pytorch_version": "2.1.2"
+            }
         ],
         "TEI": [
-                {
-                    "device": "gpu",
-                    "min_version": "1.2.1",
-                    "max_version": "1.2.3",
-                    "os_version": "ubuntu22.04",
-                    "cuda_version": "cu122",
-                    "python_version": "py310",
-                    "pytorch_version": "2.0.1"
-                },
-                {
-                    "device": "cpu",
-                    "min_version": "1.2.1",
-                    "max_version": "1.2.3",
-                    "os_version": "ubuntu22.04",
-                    "cuda_version": "cu122",
-                    "python_version": "py310",
-                    "pytorch_version": "2.0.1"
-                }
+            {
+                "device": "gpu",
+                "min_version": "1.2.1",
+                "max_version": "1.4.0",
+                "os_version": "ubuntu22.04",
+                "cuda_version": "cu122",
+                "python_version": "py310",
+                "pytorch_version": "2.0.1"
+            },
+            {
+                "device": "cpu",
+                "min_version": "1.2.1",
+                "max_version": "1.4.0",
+                "os_version": "ubuntu22.04",
+                "cuda_version": "cu122",
+                "python_version": "py310",
+                "pytorch_version": "2.0.1"
+            }
         ]
     },
     "ignore_vulnerabilities": [],
     "releases": [
         {
             "framework": "TEI",
             "device": "gpu",
-            "version": "1.2.3",
+            "version": "1.4.0",
             "os_version": "ubuntu22.04",
             "cuda_version": "cu122",
             "python_version": "py310",
             "pytorch_version": "2.0.1"
         }
```

Comment:
please add another entry for cpu image. Since we are releasing both tei cpu and gpu image altogether.
      ```
{
            "framework": "TEI",
            "device": "cpu",
            "version": "1.4.0",
            "os_version": "ubuntu22.04",
            "python_version": "py310",
            "pytorch_version": "2.0.1"
        }
```